core/templates/services/generate-content-id.service.ts

```diff
@@ -46,9 +53,7 @@ export class GenerateContentIdService {
 
   _getNextId(existingComponentIds: Array<string>,
       componentName: string): string {
-    if (componentName === AppConstants.COMPONENT_NAME_FEEDBACK ||
-        componentName === AppConstants.COMPONENT_NAME_HINT ||
-        componentName === AppConstants.COMPONENT_NAME_WORKED_EXAMPLE.QUESTION ||
+    if (componentName === AppConstants.COMPONENT_NAME_WORKED_EXAMPLE.QUESTION ||
```

Comment:
Can you explain this change? (It's probably OK but just wanted to double-check.)

Reply:
Sure- generating content_id's for feedback and hints were changed to use next_content_id_index. They no longer utilize this particular method (instead, they use getNextStateId rather than getNextId.

Reply:
OK, maybe add a note here to explain why worked examples still belong here but other components (feedback, hints, etc.) do not.